src/platform/site-wide/widgetTypes.js

```diff
@@ -62,6 +62,8 @@ export default {
   MANAGE_VA_DEBT_CTA: 'manage-va-debt-cta',
   MEDICAL_COPAYS_CTA: 'medical-copays-cta',
   MHV_SIGNIN_CTA: 'mhv-signin-cta',
+  MHV_SIMPLE_SIGNIN_CTA: 'mhv-simple-signin-cta',
+  MODERN_SECURE_MESSAGING_PAGE: 'modern-secure-messaging-page',
```

Comment:
Is there a use case for MHV_SIMPLE_SIGNIN_CTA widget?
I'm assuming CAIA will be using the MODERN_SECURE_MESSAGING_PAGE widget.